### PR TITLE
Updated README.md for GCI cohort docs task

### PR DESCRIPTION
**Pull request by f4ww4z**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,208 @@
-The purpose of the [OpenMRS Cohort Module](https://wiki.openmrs.org/display/projects/Cohort+Module) is to manage Cohort related Data, Types, Attributes, Attribute Types and Members while using REST to allow external system integration. Cohort management can be performed one of two ways:
+# Cohort Module
 
- 1. Simple cohorts for single entities
- 2. Group cohorts catering to collections like households.
+_Created by Unknown User (ahabib), last modified by Unknown User (bholagabbar) on 2016-08-25_
```
Comment (bholagabbar) [inline README.md]: remove this

File: README.md [hunk 2]
```diff
@@ -1,6 +1,208 @@
...
+* Cohort management system unlike patient management system deals specifically to monitoring and suggesting possible cures to group of cohort members under a specific cohort unlike separate diagnosis for a particular patient.
+
+* We can analyze trends and patterns and this helps in research an policy making
+
+
+## Objective of the Project
+
+* The purpose is to manage cohort related data
+
+* cohorts,
```
Comment (bholagabbar) [inline README.md]: Too many spaces between bullets. Make it cohesive

File: README.md [hunk 3]
```diff
@@ -1,6 +1,208 @@
...
+
+* cohort typ
```
Comment (bholagabbar) [inline README.md]: typo. It's 'type'

File: README.md [hunk 4]
```diff
@@ -1,6 +1,208 @@
...
+
+* The two modules Rest Module of 2.11+ version and HTML Form Entry of 2.5+ version are the required modules.
+
+* **HTML FORM ENTRY OMOD FILE**: [https://modules.openmrs.org/\#/show/58/htmlformentry](https://modules.openmrs.org/#/show/58/htmlformentry)
+
+* **REST MODULE OMOD FILE**: [https://modules.openmrs.org/\#/show/153/webservices-rest](https://modules.openmrs.org/#/show/153/webservices-rest)
+
+* These modules should be running on OpenMRS core before installing the Cohort Module.
+
+#### Video for Installation Of The Module
```
Comment (bholagabbar) [inline README.md]: Remove this section, it includes an older version. I'm gonna put up a task to demo the new cohort module

File: README.md [hunk 5]
```diff
@@ -1,6 +1,208 @@
...
+* CohortModule doesnt support Obs Group Component
+
+* The user can create a HTML form under the Manage HTML forms tab and then open the htmlForm under the module, the form created is viewed.
+
+* The parameters date, location and provider needs to be included in the form and enter the rest of the fields in the form, the cohort encounters and observations related to the form are inserted into the database
+
+* To know more about HTML Form Entry Module click here: [HTML Form Entry Module](https://wiki.openmrs.org/display/docs/HTML+Form+Entry+Module)
+
+
+## Quick Demonstration
```
Comment (bholagabbar) [inline README.md]: Remove this for now as well

File: README.md [hunk 6]
```diff
@@ -1,6 +1,208 @@
...
+
+***
+
+##### Note
```
Comment (bholagabbar) [inline README.md]: REmove

**Review by bholagabbar — CHANGES_REQUESTED** [review]
Kindly make the changes, keep the formatting neat, clean and appealing. Don't be in a hurry!

**Comment by f4ww4z** [comment]
Ok, I think I made the changes. I also improved other layout formats.

File: README.md [hunk 7]
```diff
@@ -1,6 +1,172 @@
...
+
+> The demonstration has been done on the legacy UI of this module. The current UI has been changed. However, the essential process remains the same.
+
+### Downloads and Resources
+
+* [**Source Code**](https://github.com/OpenSRP/openmrs-module-cohort)
+* **Project Pages**: [**Cohort Module**](https://wiki.openmrs.org/display/~sharonvarghese/Cohort+Module)**, **[**Cohort Module Enhacements**](https://wiki.openmrs.org/display/projects/Cohort+Module+Enhancements)
+* [**OpenMRS Talk Discussions**](https://talk.openmrs.org/search?q=cohort%20module)
+* [**Sharon's GSoC '15 Blog posts**](https://sharonvarghese1.wordpress.com/)
+* [**Shreyrans' GSoC '16 Blog posts**](http://blog.shreyanssheth.com/)
```
Comment (bholagabbar) [inline README.md]: Well you spelt my name wrong

File: README.md [hunk 8]
```diff
@@ -1,6 +1,172 @@
...
+
+* Cohort management system unlike patient management system deals specifically to monitoring and suggesting possible cures to group of cohort members under a specific cohort unlike separate diagnosis for a particular patient.
+
+* We can analyze trends and patterns and this helps in research an policy making
+
+
+## Objective of the Project
+
+* The purpose is to manage cohort related data
+* cohorts
```
Comment (bholagabbar) [inline README.md]: The cohort attributes need to be further nested like this

* Dog
    * German Shepherd
    * Belgian Shepherd
        * Malinois
        * Groenendael
        * Tervuren
* Cat
    * Siberian
    * Siamese

**Comment by f4ww4z** [comment]
sorry which lines to remove? thanks

I've updated the file based on your comments @bholagabbar .

File: README.md (hunk 8 above)
Comment (bholagabbar) [reply README.md]: @Magicpotatoes2 you did not do this

File: README.md [hunk 9]
```diff
@@ -1,6 +1,172 @@
...
+* The parameters date, location and provider needs to be included in the form and enter the rest of the fields in the form, the cohort encounters and observations related to the form are inserted into the database
+
+* To know more about HTML Form Entry Module click here: [HTML Form Entry Module](https://wiki.openmrs.org/display/docs/HTML+Form+Entry+Module)
+
+> The demonstration has been done on the legacy UI of this module. The current UI has been changed. However, the essential process remains the same.
+
+### Downloads and Resources
+
+* [**Source Code**](https://github.com/OpenSRP/openmrs-module-cohort)
+* **Project Pages**: [**Cohort Module**](https://wiki.openmrs.org/display/~sharonvarghese/Cohort+Module)**, **[**Cohort Module Enhacements**](https://wiki.openmrs.org/display/projects/Cohort+Module+Enhancements)
```
Comment (bholagabbar) [inline README.md]: Buggy markdown. Stray * symbol

File: README.md [hunk 10]
```diff
@@ -1,6 +1,172 @@
-The purpose of the [OpenMRS Cohort Module](https://wiki.openmrs.org/display/projects/Cohort+Module) is to manage Cohort related Data, Types, Attributes, Attribute Types and Members while using REST to allow external system integration. Cohort management can be performed one of two ways:
```
Comment (bholagabbar) [inline README.md:1]: Add an introductory paragraph about the cohort module, like it was before you modified the REAMDE

**Review by bholagabbar — CHANGES_REQUESTED** [review]

**Comment by f4ww4z** [comment]
I am very sorry about the half-complete task. I forgot to compare with the [Cohort wiki](https://wiki.openmrs.org/display/projects/Cohort+Module) after adding my changes based on your comments. I promise I will do better next time. I am learning to take my time also.

I have a question: may I fix spelling errors? They are just simple ones like `an` supposed to be `and` and etc. Thank you.

- A big sorry I spelt your name wrong! It's definitely not on purpose.

**Comment by bholagabbar** [comment]
@Magicpotatoes2 That's alright. Take your time learning but make sure you complete your tasks before submitting. Make as many changes as you want, make the spelling changes, correct everything and only then resubmit it

File: README.md [hunk 11]
```diff
@@ -1,6 +1,178 @@
...
```
Comment (bholagabbar) [inline README.md:1]: REST isn't needed in **bold**

File: README.md [hunk 12]
```diff
@@ -1,6 +1,178 @@
...
- 2. Group cohorts catering to collections like households.
+The purpose of the [OpenMRS Cohort Module](https://wiki.openmrs.org/display/projects/Cohort+Module) is to manage Cohort-related Data, Types, Attributes, Attribute Types and Members while using **REST** to allow external system integration. Cohort management can be performed by one of two ways:
 
-<b>The official documentation and all the relevant resources can be found [HERE](https://wiki.openmrs.org/display/projects/Cohort+Module) </b>
+1. Simple cohorts for single entities.
+2. Group cohorts catering to collections like households.
+
+## Introduction
+
+##### What's a cohort?
```
Comment (bholagabbar) [inline README.md]: Slightly bigger. #### What's a cohort is fine

**Review by bholagabbar — CHANGES_REQUESTED** [review]

**Comment by bholagabbar** [comment]
@Magicpotatoes2 This looks good. Make the small changes I told you and *Squash your commits*. This means, that you have to squash your 5 commits into 1 commit. Look up google to see how to do this (squash commits).

**Comment by f4ww4z** [comment]
@bholagabbar I've done it! Squashed **all** commits into one big one.

**Comment by bholagabbar** [comment]
@Magicpotatoes2 Good job! Why haven't you submitted the GCI task yet?

**Comment by f4ww4z** [comment]
@bholagabbar Done! :smile:

**Pull request merged**